ourjapanlife/findadoc-server
# new docker scripts

Resolves #120


# What changed
- Add docker scripts
- Rename "docker:run" script

# Testing instructions
```sh
yarn docker:build
yarn docker:run
```

Visit http://localhost:49120/ and confirm you see Apollo server 🪐
